sdk/search/azure-search-documents/azure/search/documents/_api_versions.py

```diff
@@ -11,7 +11,7 @@ class ApiVersion(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     #: this is the default version
     V2020_06_30 = "2020-06-30"
     V2023_11_01 = "2023-11-01"
-    V2024_03_01_PREVIEW = "2024-03-01-Preview"
+    V2024_05_01_PREVIEW = "2024-05-01-Preview"
```

Comment:
Noticing that the generated code is defaulting with `2024-05-01-preview` with the lowercase `P`. You know if things are case-sensitive? Still might be better to remain consistent.